agent/bench-scripts/pbench-specjbb2005

```diff
@@ -170,87 +179,207 @@ while true; do
 		;;
 	esac
 done
+if [ ! -d "${specjbb2005_dir}" ]; then
+	printf -- "$*\n\n"
+	printf "\t${benchmark}: you specified an invalid --specjbb2005-dir option (\"$specjbb2005_dir\")\n\n"
+	usage
+	exit 1
+fi
 verify_tool_group $tool_group
-
 pbench-collect-sysinfo --sysinfo=$sysinfo --check
 if [ $? != 0 ]; then
-        printf -- "$*\n\n"
-        printf "\t${benchmark}: you specified an invalid --sysinfo option (\"$sysinfo\")\n\n"
-        usage
-        exit 1
+	printf -- "$*\n\n"
+	printf "\t${benchmark}: you specified an invalid --sysinfo option (\"$sysinfo\")\n\n"
+	usage
+	exit 1
 fi
 
-if [ "$stop_warehouses" == "default" ]; then
-	stop_warehouses=`echo "$nr_cpus *2 /$nr_jvms" | bc`
+if [ "${nr_jvms}" == "node" ]; then
+	# Find out number of numa nodes
+	nr_jvms=$(ls -1d /sys/devices/system/node/node* | wc -l)
+fi
+if [ "${nr_jvms}" -eq "${nr_jvms}" ] 2>/dev/null ; then
```

Comment:
I was a bit baffled by this: I guess it (and the subsequent ones) tests that the variable's value is a number. If so, I think it deserves a comment. If not, it certainly deserves a comment!

Reply:
Sure I'll add comments.

Reply:
Added comments.